### PR TITLE
feat: register & get  CustomClaimType, allow CustomClaim

### PR DESCRIPTION
**Pull request by sansan**
### JIRA Link 

https://polymesh.atlassian.net/browse/DA-908

### Changelog / Description 

Allows registering and getting customClaimType and issuing CustomClaim

### Checklist - 

- [x] New Feature ?
- [x] Updated swagger annotation (if API structure is changed) ?
- [x] Unit Test (if possible) ?
- [x] Updated the Readme.md (if required) ?

### CODE REVIEW

File: src/claims/claims.controller.ts [hunk 1]
```diff
@@ -77,4 +80,64 @@ export class ClaimsController {
...
+  ): Promise<TransactionResponseModel> {
+    const serviceResult = await this.claimsService.registerCustomClaimType(args);
+
+    return handleServiceResult(serviceResult);
+  }
+
+  @ApiOperation({
+    summary: 'Get CustomClaimType',
+    description:
+      'This endpoint will retrieve the custom claim type by id or name. <br /> `ID` or name needs to be provided. <br/> If both are provided will fetch using `ID`',
```
Comment (polymath-eric) [inline src/claims/claims.controller.ts:110]:
```suggestion
      'This endpoint will retrieve the custom claim type by id or name. <br /> ID or name needs to be provided. <br/> If both are provided will fetch using ID',
```

(nit) when I see the backticks I read it as a literal thing I should copy paste. 

If using them I'd write `id` or `name`, but I don't think they are needed.

File: src/claims/claims.controller.ts [hunk 2]
```diff
@@ -77,4 +80,64 @@ export class ClaimsController {
...
+  })
+  @ApiQuery({
+    name: 'id',
+    description: 'The id of the CustomClaimType',
```
Comment (polymath-eric) [inline src/claims/claims.controller.ts:114]:
```suggestion
    description: 'The ID of the CustomClaimType',
```

**Review by polymath-eric — APPROVED** [review]
LGTM, two minor nits on the doc comments

**Review by VictorVicente — APPROVED** [review]

**Pull request merged**